tp/src/admin/no_op.rs

```diff
@@ -0,0 +1,30 @@
+// Copyright 2021 Cargill Incorporated
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! An implementations of `AdminPermission` that always returns true
+
+use sawtooth_sdk::processor::handler::ApplyError;
+
+use crate::state::SabreState;
+
+use super::AdminPermission;
+
+#[derive(Default)]
+pub struct NoopAdminPermission;
+
+impl AdminPermission for NoopAdminPermission {
```

Comment:
Should the name of this be AllowAll? I would imagine No-Op makes sense if the return value was `()` or `Result<(), _>`, but this is making a decision on the result.

Reply:
I am open to all naming suggestions.

Reply:
ill update to allow